integration-cli/docker_cli_info_test.go

```diff
@@ -26,12 +28,13 @@ func (s *DockerSuite) TestInfoEnsureSucceeds(c *check.C) {
 		"CPUs:",
 		"Total Memory:",
 		"Kernel Version:",
-		"Storage Driver:"}
+		"Storage Driver:",
+		fmt.Sprintf("Experimental: %t", utils.ExperimentalBuild()),
```

Comment:
hmm ok, minor detail, but would prefer not have the word "experimental" in a "stable" release